package.json

```diff
@@ -36,18 +36,19 @@
     "react-dom": ">=0.14.7 <=16.x"
   },
   "devDependencies": {
+    "@babel/cli": "^7.1.5",
+    "@babel/core": "^7.1.5",
+    "@babel/plugin-proposal-class-properties": "^7.1.0",
+    "@babel/plugin-proposal-do-expressions": "^7.0.0",
```

Comment:
Think the `do` is only used in one place in the code, is it worth the boilerplate (no idea what babel transpiles)